pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -529,8 +529,9 @@ def make_package(args):
         aars=aars,
         jars=jars,
         android_api=android_api,
-        build_tools_version=build_tools_version
-        )
+        build_tools_version=build_tools_version,
+        is_library=get_bootstrap_name() == 'service_library',
```

Comment:
This also looks like a candidate for a setting on the bootstrap class

Reply:
It's set up this way because the dist is nominally independent of the main p4a code, although that's an old design decision that isn't really true any more. So, this is fine for now, but I agree it would be nice to consolidate the whole api.